aptos-move/framework/src/natives/transaction_context.rs

```diff
@@ -405,6 +424,10 @@ pub fn make_all(
             "multisig_payload_internal",
             native_multisig_payload_internal,
         ),
+        (
```

Comment:
you need to add now:

("unique_session_hash", native_get_txn_hash),

so that second stage can be done

and you can rename native_get_txn_hash to native_unique_session_hash in rust for cleanliness, as this is just rust code

Reply:
Renamed `native_get_txn_hash` to `native_unique_session_hash`,
and added `("unique_session_hash", native_unique_session_hash),` and
`("raw_transaction_hash", native_raw_transaction_hash_internal,),`

@igor-aptos , by the way, we already have some existing native functions such as `gas_payer_internal` that has its public wrapper `gas_payer`.
We can register `gas_payer` as native in this release, right? So, adding,
```
("gas_payer", native_gas_payer_internal),
```

Reply:
yup, if you want to clean things up and optimize out - we can do that.

if there is any chance we are going to need to revert a flag, we shouldn't remove the internal variant :)